src/drivers/safety_button/SafetyButton.hpp

```diff
@@ -71,15 +73,17 @@ class SafetyButton : public ModuleBase<SafetyButton>, public px4::ScheduledWorkI
 
 private:
 
-	void CheckButton();
+	bool CheckButton();
```

Comment:
I would rename this method to `isButtonPressed()` or `hasButtonBeenPressed()` to communicate what a true return value means.